health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/AddOrEditMapping.js

```
@@ -1,196 +1,217 @@
-    }));
 
-    const renderInput = (column) => {
+  const renderInput = (column) => {
+    if ((column?.description === "Boundary Code" || column?.description === "Boundary Code (Mandatory)") && typeOfOperation === "edit") {
+      return null; // Hide boundary selection in edit mode
+    }
+    if (column?.type === "number") {
+      return (
+        <div key={column.name} style={{ marginBottom: "1rem" }}>
+          <div style={{ display: "flex" }}>
+            <label>{column.description}</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)